nibiru/sdk.py

```diff
@@ -47,19 +47,14 @@ class Sdk:
         )
     """
 
-    query: GrpcClient
-    network: Network
-    tx: BaseTxClient
-    tx_config: TxConfig
```

Comment:
These are just type hints. Putting them next to the class docstring just makes the type hint available in functions besides `__init__`.